### PR TITLE
doc: initial cut at collaboration network proposal

### PR DESCRIPTION
**Pull request by mhdawson**
First cut at collaboration network proposal

Changes needed to docs outside the proposal are in: https://github.com/openjs-foundation/cross-project-council/pull/547

### CODE REVIEW

**Comment by mhdawson** [comment]
First cut at what the changes might look like.

I've marked do not land as the changes to the Charter, main README.md etc are outside of the proposal directory but I was not sure how else to show the complete picture.  There are also still a few missing pieces but hopefully enough to give people a better view of the proposal.

**Comment by designMoreWeb** [comment]
LGTM Great work @mhdawson

File: README.md [hunk 1]
```diff
@@ -61,6 +61,12 @@ The following projects are official OpenJS Foundation projects. If you are inter
 * [Fastify](https://fastify.io)
 * [nvm](http://nvm.sh/)
 
+
+### OpenJS Collaboration spaces
+
+The following are official OpenJS Collaboration sapces. If you are interested in fostering collaboration with the
```
Comment (dmethvin) [inline README.md]:
```suggestion
The following are official OpenJS Collaboration spaces. If you are interested in fostering collaboration with the 
```

File: README.md [hunk 2]
```diff
@@ -121,7 +132,7 @@ be excluded from future CPC meetings based on a standard CPC motion.
 
 ## Policy Change Proposal Process
 
-The OpenJS CPC is chartered to oversee the technical governance of all OpenJS Projects and Working Groups under the OpenJS Foundation. The CPC establishes the default governance, conduct, and licensing policies for all Projects. Projects have broad powers of self-governance.
+The OpenJS CPC is chartered to oversee the technical governance of all OpenJS Projecta, Collaboration spacess and Working Groups under the OpenJS Foundation. The CPC establishes the default governance, conduct, and licensing policies for all Projects and Collaboration spaces. Projects have broad powers of self-governance.
```
Comment (dmethvin) [inline README.md]:
```suggestion
The OpenJS CPC is chartered to oversee the technical governance of all OpenJS Projects, Collaboration spaces and Working Groups under the OpenJS Foundation. The CPC establishes the default governance, conduct, and licensing policies for all Projects and Collaboration spaces. Projects have broad powers of self-governance.
```

**Comment by dylans** [comment]
what's here so far looks good to me.

**Comment by mhdawson** [comment]
I've read through the CNCF SIG's and they are more along the lines of Working groups.  General literature also seems to refer to SIG's a sub-groupings of members of an organization on a topic of interest.  

What I had in mind with collaboration spaces was more around trying to provide spaces for people in the JS Ecosystem but not part of the Foundation (nor necessarily wanting to join or be part of the running of the CPC/foundation) to get together and collaborate.

A long way of saying have done a bit of reading, my preference would be to stick with the terminology outline in the PR.

**Comment by mhdawson** [comment]
@openjs-foundation/cpc I'd like to propose that this lands (minus the external governance changes) as stage 0 unless there are objections?  I would then create a PR moving it to stage 1, and re-include the governance changes so that they are still visible/reviewable.

**Comment by mhdawson** [comment]
A number of approvals, no objections and I believe we agreed this could land as stage 0 in the CPC meeting this week.

**Pull request merged**